lib/reload.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(app) {
```

Comment:
If we are keeping this module, it should have `//reload` at the top as a comment to keep this consistent with the rest of roosevelt